templates/pages/listing.html

```diff
@@ -1,218 +1,63 @@
+        {% if entry.image %}
+        <a href="{{url_for('entry', entry_id = entry._id)}}"><img src="{{ entry.image }}" class="card-img-top"
                 alt="alt text"></a>
+        {% endif %}
         <div class="card-body">
             <h5 class="card-title">
+                {% if entry.is_fav %}
                 <span class="material-icons fav">
                     favorite
-                </span> <a href="{{url_for('entry')}}">Reviewed Item</a></h5>
-            <p class="card-text">This is the short version of the description but
-                there is a lot more to be seen in the view itself.</p>
-            <p class="card-text" aria-hidden="true">Rating:
-                <span class="rating">
-                    <span class="material-icons">
-                        star
-                    </span>
-                    <span class="material-icons">
-                        star
-                    </span>
-                    <span class="material-icons">
-                        star
-                    </span>
-                    <span class="material-icons">
-                        star_half
-                    </span>
-                    <span class="material-icons">
-                        star_outline
-                    </span>
                 </span>
-            </p>
-            <span class="sr-only">Rating: 4/5</span>
-            <p class="card-text">
-                <small>Tags: </small>
-                <a href="{{url_for('listing')}}"><span class="badge badge-pill badge-primary">food</span></a>
-                <a href="{{url_for('listing')}}"><span class="badge badge-pill badge-primary">drink</span></a>
-                <a href="{{url_for('listing')}}"><span class="badge badge-pill badge-primary">yogurt</span></a></p>
-        </div>
-        <div class="card-footer">
-            Last updated on 15/06/20
-            <span class="material-icons delete-btn">delete</span>
-
-        </div>
+                {% endif %}
 
+                <a href="{{url_for('entry', entry_id = entry._id)}}">{{ entry.name }}</a>
+            </h5>
+            {% if entry.image %}
+            <p class="card-text">{{ entry.description|truncate(75, end='...') }}</p>
+            {% else %}
+            <p class="card-text">{{ entry.description|truncate(150, end='...') }}</p>
+            {% endif %}
 
-    </div>
-
-    <div class="card">
-        <a href="{{url_for('entry')}}"><img src=" http://placeimg.com/640/480/any" class="card-img-top"
-                alt="alt text"></a>
-        <div class="card-body">
-            <h5 class="card-title">
-                <a href="{{url_for('entry')}}">Reviewed Item Unfavorited</a></h5>
-            <p class="card-text">This is the short version of the description but
-                there is a lot more to be seen in the view itself.</p>
-            <p class="card-text">Rating:
+            <p class="card-text" aria-hidden="true">Rating:
                 <span class="rating">
```

Comment:
This should be `aria-hidden="true"` so it is not read out loud by screen-readers.